modules/dynamic-plugins/rhdh-supported-plugins.adoc

```diff
@@ -15,75 +15,219 @@ Additional detail on how Red Hat provides support for bundled community dynamic
 
 There are 61 plugins available in {product}. See the following table for more information:
 
-[dynamic-plugins-matrix]
-.Dynamic plugins support matrix
+== Red Hat / Partner provided plugins
 
-[%header,cols=8*]
+Red Hat and its partners provide the following fully supported plugins.
+
+[%header,cols=5*]
 |===
-|*Name* |*Plugin* |*Role* |*Version* |*Support Level*
-|*Path* |*Required Variables* |*Default*
-|3scale  |@janus-idp/backstage-plugin-3scale-backend |Backend |1.6.0 |Red Hat Tech Preview
+|*Name* |*Plugin* |*Version* |*Path and required variables* |*Default* 
+|Analytics Provider Segment  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-analytics-provider-segment/v/1.5.0[@janus-idp/backstage-plugin-analytics-provider-segment] |1.5.0 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-analytics-provider-segment
+
+`SEGMENT_WRITE_KEY`
+
+`SEGMENT_TEST_MODE`
+
+|Enabled
+
+|Argo CD  - Frontend |https://npmjs.com/package/@janus-idp/backstage-plugin-argocd/v/1.2.3[@janus-idp/backstage-plugin-argocd] |1.2.3 
+|./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd
```

Comment:
All the plugins path should be formatted as code.
```suggestion
|`./dynamic-plugins/dist/janus-idp-backstage-plugin-argocd`
```

Reply:
ok, I can change this... it wasn't like that in 1.2 

https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html-single/configuring_plugins_in_red_hat_developer_hub/index#rhdh-supported-plugins

Do you want the name/URL also formatted as code?

Reply:
Yes @nickboldt, please change the name/URL to formatted as code.